plugins/org.eclipse.n4js/src/org/eclipse/n4js/binaries/BinariesConstants.java

```diff
@@ -106,7 +106,7 @@
 	/** The label for {@code Node.js}. */
 	public static final String NODE_LABEL = "Node.js";
 	/** The {@code Node.js} binary name (OS specific). */
-	public static final String NODE_BINARY_NAME = isWindows() ? "node.exe" : "node";
+	public static final String NODE_BINARY_NAME = "node";
```

Comment:
Maybe rename to `NODE_BINARY_BASE_NAME` and add note to comment that this name does *not* include any file extensions.